Extension/src/Debugger/configurationProvider.ts

```diff
@@ -593,13 +593,102 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
         return false;
     }
 
+    private getLaunchJsonPath(): string | undefined {
+        return util.getJsonPath("launch.json");
+    }
+
+    public getRawLaunchJson(): Promise<any> {
+        const path: string | undefined = this.getLaunchJsonPath();
+        return util.getRawJson(path);
```

Comment:
I believe this deletes any comments in the `launch.json` file.  I would think we should try to preserve the comments.  Preserving formatting would be nice too, but using the VSCode setting for tab size should be ok.

Reply:
Could you add a comment to the code here that references the VS Code issue 29453?

Reply:
@Colengc currently we are not preserving the comments, right?